Lib/test/test_gc.py

```diff
@@ -1080,28 +1091,31 @@ def make_ll(depth):
                 head = LinkedList(head, head.prev)
             return head
 
-        head = make_ll(10000)
-        count = 10000
+        head = make_ll(1000)
+        count = 1000
 
-        # We expect the counts to go negative eventually
-        # as there will some objects we aren't counting,
-        # e.g. the gc stats dicts. The test merely checks
-        # that the counts don't grow.
+        # There will some objects we aren't counting,
+        # e.g. the gc stats dicts. This test checks
+        # that the counts don't grow, so we try to
+        # correct for the uncounted objects
+        # This is just an estimate.
+        CORRECTION = 20
```

Comment:
What's the likelihood that this number might need to be changed in the future?  Keep in mind that readers that aren't as familiar with GC (especially the incremental version) will be hard-pressed to know which objects aren't counted (nor how many there might be).  Any additional information you can leave in this note would be supremely helpful.

Reply:
As it says, it is just a estimate. 
I added this in an attempt to make the test stricter, so it is more likely to catch a regression, than anything exact.